VideoWeb/VideoWeb/Controllers/InternalEventController.cs

```diff
@@ -171,5 +177,25 @@ public async Task<IActionResult> AllocationHearings(AllocationHearingsToCsoReque
                 return StatusCode(e.StatusCode, e.Response);
             }
         }
+        
+        [HttpPost("HearingCancelled")]
+        [SwaggerOperation(OperationId = "HearingCancelled")]
+        [ProducesResponseType((int)HttpStatusCode.NoContent)]
+        [ProducesResponseType(typeof(string), (int)HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> HearingCancelled(Guid conferenceId)
+        {
+            await _hearingCancelledEventNotifier.PushHearingCancelledEvent(conferenceId);
+            return NoContent();
+        }
+
+        [HttpPost("HearingDetailsUpdated")]
+        [SwaggerOperation(OperationId = "HearingDetailsUpdated")]
+        [ProducesResponseType((int)HttpStatusCode.NoContent)]
+        [ProducesResponseType(typeof(string), (int)HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> HearingDetailsUpdated(Guid conferenceId)
+        {
+            await _hearingDetailsUpdatedEventNotifier.PushHearingDetailsUpdatedEvent(conferenceId);
```

Comment:
Good opportunity to force a refresh of the cache here, and then publish the conference object? That way the clients do not have to make an api call to get the object again

Reply:
Updated it to force refresh the cache. I agree about publishing the conference object, but not sure we need it yet, as currently the event only needs to act as a trigger to refresh the list on the client, we're not using the contents of the conference object. Maybe later down the line we can hook this up to the redux store and use the conference objects then